src/lib.rs

```diff
@@ -763,10 +763,7 @@ impl Document {
     }
 
     fn serialize_opts() -> SerializeOpts {
-        SerializeOpts {
-            traversal_scope: TraversalScope::ChildrenOnly,
```

Comment:
Is this the default now ?

Reply:
Looks like it was, even before: https://github.com/servo/html5ever/commit/015cfb029be4d6404ec0cc3e68186cb82bd47db6.